integration-tests/qute/src/main/java/org/apache/camel/quarkus/component/qute/it/QuteRoute.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2020, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.apache.camel.quarkus.component.qute.it;
+
+import org.apache.camel.builder.RouteBuilder;
+
+public class QuteRoute extends RouteBuilder {
+    @Override
+    public void configure() throws Exception {
+        from("direct:test")
+                .to("qute:file:target/classes/hello.txt");
```

Comment:
Could you please switch to example to work with `classpath:` URI so that the itest is runnable outside Camel Quarkus source tree without any additional setup?

Reply:
I use the `qute:hello.txt` but this is failing when running with the native image which can not find the resource.

Reply:
That can be fixed, see other extensions how they do it.

Reply:
can you point it out ? I just search the extensions but could not get the hint. Thanks !